src/recipe.rst

```diff
@@ -19,6 +19,29 @@ Your final recipe should have no comments and follow the order in the example.
 *If there are details you are not sure about please open a pull request. The conda-forge team will be happy to answer your questions.*
 
 
+In case you build your first recipe using conda-forge, the we provide here a step-by-step instruction and checklist that might help you with a sucessfully build
+
+Step-by-step instructions
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. fork the `example recipes <https://github.com/conda-forge/staged-recipes/tree/master/recipes>`_
+2. within your forked copy, generate a new folder in the recipes subdirectory and copy meta.yml from the example directory. Leave the example directory unchanged!
+3. edit the example recipe (`meta.yml <https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml>`_) as you need it
+4. generate the ssh 256 key like described in the example recipe using the
+   openssl tool
+5. ensure to fill in the `tests` section. Simple tests would just import your module. More advanced would be that you run the tests coming with your package itself. For this you would need to TBD SECTION ON TESTING 
+6. remove all comments in the `meta.yml <https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml>`_
+
+
+Checklist
+~~~~~~~~~
+
+* ensure that the license and license family descriptors have the right case and that the license is correct. Note that case sensitive inputs are required (e.g. Apache 2.0 and not APACHE 2.0).
```

Comment:
Only the license metadata is required. The license family is optional. Unclear at this time if `conda-build` will keep supporting that or not.